### PR TITLE
style: improve code readability

### PR DESCRIPTION
**Pull request by hitzhangjie**
improve readability of logging and error, func and var names, comments, logic, etc.

### CODE REVIEW

File: store/store.go [hunk 1]
```diff
@@ -59,12 +59,12 @@ func RegisterStore(s Store) error {
...
 	name := config.Name
 	if name == "" {
-		return nil, errors.New("store Name is empty")
+		return nil, errors.New("store name is empty")
 	}
 
 	store, ok := StoreSlots[name]
 	if !ok {
-		return nil, errors.New("no such name Store")
+		return nil, errors.New("store not found")
```
Comment (andrewshan) [inline store/store.go:67]: it's better to give the name to the not found store

File: store/store.go [hunk 2]
```diff
@@ -84,7 +84,7 @@ func SetStoreConfig(conf *Config) {
 func initialize(s Store) {
 	once.Do(func() {
 		if err := s.Initialize(config); err != nil {
-			fmt.Printf("store do initialize err: %s", err.Error())
+			fmt.Printf("[ERROR] initialize store fail: %v", err)
```
Comment (andrewshan) [inline store/store.go:87]: give the store name in the log

**Comment by andrewshan** [comment]
@hitzhangjie hello, can you resolve the conflict in this PR?

File: store/store.go (hunk 2 above)
Comment (hitzhangjie) [reply store/store.go]: done

File: store/store.go (hunk 1 above)
Comment (hitzhangjie) [reply store/store.go]: done

**Pull request merged**